WORKFLOW.md

```diff
@@ -0,0 +1,69 @@
+# Grafana workflow
+
+This document is based on [GOVERNANCE.md](GOVERNANCE.md). We assume good faith and intend to keep all processes as lightweight as possible but as specific as required. In case of disagreements about anything in this document, GOVERNANCE.md applies.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+Git and [GitHub terminology](https://help.github.com/en/github/getting-started-with-github/github-glossary) are used throughout this document.
+
+Team members and their access to repositories is maintained through [GitHub teams](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams). Team maintainers add and remove team members as per GOVERNANCE.md
+
+
+# Code changes
```

Comment:
```suggestion
## Code changes
```

Reply:
Please remove the empty line above this paragraph.

Reply:
Adjust the heading levels that follow this to be +1 in depth until the next h1 to h2 adjustment.

Reply:
Newline removed, heading is how I think the structure makes sense.